src/main/java/uk/gov/hmcts/reform/sscs/model/SessionCaseCodeMapping.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SessionCaseCodeMapping {
```

Comment:
This might be useful:
https://github.com/hmcts/sscs-common/blob/1434b27bbd68ff309dae70c76878c8cd33bcbccd/src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/CaseSubType.java